app/src/main/java/com/adevinta/mappicker/MainActivity.kt

```diff
@@ -179,3 +128,168 @@ class MainActivity : AppCompatActivity() {
+        // .withGeolocApiKey("<PUT API KEY HERE>")
+        // .withGooglePlacesApiKey("<PUT API KEY HERE>")
+        .withSearchZone("es_ES")
+        // .withSearchZone(SearchZoneRect(LatLng(26.525467, -18.910366), LatLng(43.906271, 5.394197)))
+        .withDefaultLocaleSearchZone()
+        // .shouldReturnOkOnBackPressed()
+        // .withStreetHidden()
+        // .withCityHidden()
+        // .withZipCodeHidden()
+        // .withSatelliteViewHidden()
+        .withGoogleTimeZoneEnabled()
+        // .withVoiceSearchHidden()
+        .withUnnamedRoadHidden()
+        // .withSearchBarHidden()
+        .build(activity)
+
+    // this is optional if you want to return RESULT_OK if you don't set the
+    // latitude/longitude and click back button
+    locationPickerIntent.putExtra("test", "this is a test")
+
+    activity.startActivityForResult(locationPickerIntent, MAP_BUTTON_REQUEST_CODE)
+}
+
+private fun onLegacyMapClicked(context: Context) {
+    val activity = context as Activity
+    val locationPickerIntent = LocationPickerActivity.Builder()
+        .withLocation(DEMO_LATITUDE, DEMO_LONGITUDE)
+        .withUnnamedRoadHidden()
+        .withLegacyLayout()
+        .build(activity)
+    activity.startActivityForResult(locationPickerIntent, MAP_BUTTON_REQUEST_CODE)
+}
+
+private val lekuPois: List<LekuPoi>
+    get() {
+        val pois = ArrayList<LekuPoi>()
+
+        val locationPoi1 = Location("leku")
+        locationPoi1.latitude = POI1_LATITUDE
+        locationPoi1.longitude = POI1_LONGITUDE
+        val poi1 = LekuPoi(UUID.randomUUID().toString(), "Los bellota", locationPoi1)
+        pois.add(poi1)
+
+        val locationPoi2 = Location("leku")
+        locationPoi2.latitude = POI2_LATITUDE
+        locationPoi2.longitude = POI2_LONGITUDE
+        val poi2 = LekuPoi(UUID.randomUUID().toString(), "Starbucks", locationPoi2)
+        poi2.address = "Plaça de la Sagrada Família, 19, 08013 Barcelona"
+        pois.add(poi2)
+
+        return pois
+    }
+
+private fun onMapPoisClicked(context: Context) {
+    val activity = context as Activity
+    val locationPickerIntent = LocationPickerActivity.Builder()
+        .withLocation(DEMO_LATITUDE, DEMO_LONGITUDE)
+        .withPois(lekuPois)
+        .build(activity)
+
+    activity.startActivityForResult(locationPickerIntent, MAP_POIS_BUTTON_REQUEST_CODE)
+}
+
+private fun onMapWithStylesClicked(context: Context) {
+    val activity = context as Activity
+    val locationPickerIntent = LocationPickerActivity.Builder()
+        .withLocation(DEMO_LATITUDE, DEMO_LONGITUDE)
+        .withMapStyle(R.raw.map_style_retro)
+        .build(activity)
+    activity.startActivityForResult(locationPickerIntent, MAP_POIS_BUTTON_REQUEST_CODE)
+}
+
+@Composable
+@Preview(showBackground = true)
+fun MainView() {
+    val context = LocalContext.current
+
+    Column(Modifier.padding(16.dp, 0.dp, 16.dp, 0.dp).fillMaxSize()) {
+        Spacer(modifier = Modifier.size(20.dp))
```

Comment:
You could replace all the `Spacer(modifier = Modifier.size(20.dp))` with a parameter in the parent Column: 
```
Column(
  Modifier
    .padding(horizontal = 16.dp)
    .fillMaxSize(),
  verticalArrangement = Arrangement.spacedBy(20.dp)
) {

Reply:
Many thanks for your suggestion!